### PR TITLE
Update A4T section to provide further clarification on server and client side logging

### PR DESCRIPTION
**Pull request by swarna04**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/adobe-journey-optimizer-decisioning/index.md [hunk 1]
```diff
@@ -231,13 +231,15 @@ After you finish your rule configuration, verify the rule details are as shown b
...
+
+<InlineAlert variant="info" slots="text"/>
 
-For this integration to work, make sure Analytics is enabled in your datastream configuration for the desired environment and Report Suite information is provided.
+Server-side logging: If Analytics is enabled and configured in your datastream for the desired environment, then it is considered server-side logging. In this case, the Experience Edge handles forwarding any Target A4T payloads to Adobe Analytics, upon tracking method calls, and no analytics tokens are returned to the client.<br/>Client-side logging: If Analytics is disabled in your datastream for the desired environment, then it is considered client-side logging. In this case, analytics tokens are returned to the client and it is the responsibility of the customer to extract and send the data to Adobe Analytics, if desired.
```
Comment (calebk1) [inline src/pages/documentation/adobe-journey-optimizer-decisioning/index.md:242]: For Analytics tokens - are these tokens from Adobe Analytics? If it is, it should be "... and no Analytics tokens are returned..." and "... this case, Analytics tokens are returned..."

**Review by calebk1 — APPROVED** [review]
Looks good! Just a small comment.

**Review by sbenedicadb — APPROVED** [review]

File: src/pages/documentation/adobe-journey-optimizer-decisioning/index.md (hunk 1 above)
Comment (emdobrin) [reply src/pages/documentation/adobe-journey-optimizer-decisioning/index.md:242]: nit: can we bold "Server-side logging" and "Client-side logging"

**Review by emdobrin — APPROVED** [review]
thanks for the updates @swarna04 🎉

File: src/pages/documentation/adobe-journey-optimizer-decisioning/index.md (hunk 1 above)
Comment (swarna04) [reply src/pages/documentation/adobe-journey-optimizer-decisioning/index.md:242]: @calebk1 The tokens returned are meant for AA, I'll fix it.

**Pull request merged**